### ISSUE TITLE
Animal sniffer 1.14 doesn't detect java version

### ISSUE THREAD

**Issue by GerardPaligot** [mojohaus/animal-sniffer#13]
When I specify 1.8 for the Java version used by my project to compile sources, animal sniffer doesn't detect that the java version isn't compatible with Java 1.6, the version checked by animal sniffer.

This is my configuration:

``` xml
<plugin>
  <groupId>org.codehaus.mojo</groupId>
  <artifactId>animal-sniffer-maven-plugin</artifactId>
  <version>1.14</version>
  <configuration>
    <signature>
      <groupId>org.codehaus.mojo.signature</groupId>
      <artifactId>java16</artifactId>
      <version>1.0</version>
    </signature>
  </configuration>
  <executions>
    <execution>
      <id>ensure-java-1.6-class-library</id>
      <phase>test</phase>
      <goals>
        <goal>check</goal>
      </goals>
    </execution>
  </executions>
</plugin>
```

But when I use a version before 1.14, I got an `IllegalArgumentException` from animal sniffer and the execution of the plugin stop the execution of my project. This is what I want (I would like more info about what doesn't work but this is a good start).

**Comment by rfscholte** [comment]
Don't think this is a bug. It just means that you're not using anything specific introduced with Java7 or Java8.

**Comment by sco0ter** [comment]
You may need to set move the configuration section into the execution section.

**Comment by slawekjaranowski** [comment]
Please retest with the latest version.
